app/controllers/admin/people_controller.rb

```diff
@@ -33,6 +35,10 @@ def update
     end
   end
 
+  def confirm_destroy
+    redirect_to [:admin, @person], alert: "Cannot destroy a person with appointments" and return unless @person.destroyable?
+  end
```

Comment:
This isn't very clear to read. Can we do the following:

```suggestion
  def confirm_destroy
    redirect_to([:admin, @person], alert: "Cannot destroy a person with appointments") unless @person.destroyable?
  end
```